ClosedXML/ClosedXML
# Load and copy images on worksheets.

#### What's this PR do?
Load existing images on worksheets and copy those images when copying worksheets.

#### Where should the reviewer start?
Most work is in `ClosedXML/Excel/XLWorkbook_Load.cs`.  Also had to fix up some things while saving to get the excel file to stay consistent when loading/saving.
 
#### How should this be manually tested?
Load up an excel with images and use `Worksheet.CopyTo` and then save and verify the worksheets look similar.

#### Any background context you want to provide?
There may be some rounding issues when converting to/from emu for the offsets, etc.  This is why I re-added the excel files for the tests to get the numbers fixed.  

#### Screenshots (if appropriate)
#### Questions:
- Is there a blog post?
- Does the knowledge base need an update?
- Does this add new (C#) dependencies?
